pysensors/utils/_constraints.py

```diff
@@ -185,10 +490,10 @@ def get_coordinates_from_indices(idx,info):
     if isinstance(info,tuple):
         return np.unravel_index(idx,info,'F')
     elif isinstance(info,pd.DataFrame):
-        x = info.loc[idx,'X (m)']#.values
+        x = info.loc[idx,'X (m)']#.values  ## This will not always be X (m) or Y (m) CHANGE!!!
```

Comment:
`get_coordinates_from_indices` is called from BaseConstraint, so this needs to be fixed, or remove calling it from BaseConstraint